tests/UI/Base.php

```diff
@@ -50,6 +51,21 @@ public function txt($a_topic, $a_default_lang_fallback_mod = "") {
 	}
 }
 
+class LoggingJavaScriptBinding implements JavaScriptBinding {
+	private $count = 0;
+	public $ids = array();
+	public function createId() {
+		$this->count++;
+		$id = "id_".$this->count;
```

Comment:
I just noticed the test does not use randomized ids yet - but I guess this is not a problem.